mods/ra/maps/evacuation/map.yaml

```diff
@@ -0,0 +1,1757 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: Evacuation
+
+Author: Scott_NZ
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 16,16,96,96
+
+Visibility: MissionSelector, Lobby
```

Comment:
These are Co-Op mission which can be played single-player as well so this entry is valid. Our UI is just not capable enough. There is even a TODO in the code where it crashes.